ideascube/mediacenter/migrations/0013_escape_summary.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+from django.utils.html import escape
+
+
+def escape_summary(app, schema_editor):
+    Document = app.get_model('mediacenter', 'Document')
+    db_alias = schema_editor.connection.alias
+    for document in Document.objects.using(db_alias).all():
+        document.summary = "<p>{}</p>".format(escape(document.summary))
```

Comment:
> I don't see a admin going throw all the Document/Content/Book to just press edit/validate to fix this.

Me neither. Realistically, they won't have to, because things will just be displayed fine.

> There is no problem to wrap the text in <p>...</p>

There are many things which could go wrong.

For example, wrapping a `<div>` in a `<p>` is invalid HTML, and could render badly.

A `<p>` tag can only contain [phrasing content](https://www.w3.org/TR/html5/dom.html#phrasing-content-1), not any other tag.

In addition, I insist that you're not doing the same as TinyMCE in this migration:

* TinyMCE would only wrap non-block root elements in a `<p>`
* you're wrapping everything unconditionally, without looking at what it is

Who knows what could go wrong? You might "cripple the editor". 😃